Core/include/Acts/TrackFitting/detail/GsfActor.hpp

```diff
@@ -32,6 +32,23 @@
 namespace Acts {
 namespace detail {
 
+/// Small Helper class that allows to carry a tmporary value until we decide to
+/// update the actual value. The temporary value is deliberatly only accessable
+/// with a mutual reference
+template <typename T>
+class Updatable {
+  T m_tmp{};
+  T m_val{};
+
+ public:
+  Updatable() : m_tmp(0), m_val(0) {}
```

Comment:
You assume `T` can be initialized from `0`?

Reply:
I only gave `double` and `std::size_t` currently, and this is not meant to bd overly generic